randovania/games/am2r/layout/am2r_configuration.py

```diff
@@ -58,4 +66,16 @@ def unsupported_features(self) -> list[str]:
         if self.artifacts.required_artifacts > self.artifacts.placed_artifacts:
             result.append("The amount of required DNA cannot be higher than the total amount of placed DNA.")
 
+        if self.darkness_min > self.darkness_max:
+            result.append("The minimum darkness value cannot be higher than the maximum darkness value.")
```

Comment:
Add some test for these cases please

Reply:
added
Also removed the second case here, cause it felt unnecessary.